repository-data/webfiles/src/main/resources/site/freemarker/common/coronavirus-banner.ftl

```diff
@@ -5,7 +5,9 @@
 <#if showBanner?? && showBanner>
     <!--googleoff: index-->
     <article>
-        <header class="coronavirus-banner" aria-labelledby="message-summary-title" role="alert" tabindex="-1">
+        <header class="coronavirus-banner" aria-labelledby="message-summary-title" tabindex="-1">
+            <div role="alert">
+            <h2>&nbsp;</h2>
```

Comment:
I think this line can be removed?

Reply:
Yes, I concur, I have addressed this, I have removed the confusing role alert and this will be addressed with ticket DW-2702.